guides/release/routing/query-params.md

```diff
@@ -30,10 +30,10 @@ as one of `controller:articles`'s `queryParams`:
 ```javascript {data-filename=app/controllers/articles.js}
 import Controller from '@ember/controller';
 
-export default Controller.extend({
-  queryParams: ['category'],
-  category: null
-});
+export default class ArticlesController extends Controller {
+  queryParams = ['category'];
```

Comment:
I _assume_ this is ok, but we might want to get confirmation that defining the `queryParams` property on each controller instance (i.e., each new singleton as we go through an acceptance test suite) is appropriate. The syntax being replaced defines this property on the controller's prototype.